web/src/stream_settings_ui.js

```diff
@@ -508,6 +513,7 @@ export function get_left_panel_params() {
     const params = {
         input,
         show_subscribed: stream_ui_updates.show_subscribed,
+        show_not_subscribed: stream_ui_updates.show_not_subscribed,
```

Comment:
Maybe we can just store the selected name in a single variable instead of using two boolean variables, but I am not sure.

Let us keep it as it is for now and wait for further feedback.

Reply:
I think it's probably fine, since they are somewhat independent variables.